example/build.gradle

```diff
@@ -13,6 +13,10 @@ android {
     compileSdkVersion 25
     buildToolsVersion "25.0.0"
 
+    defaultConfig {
+        minSdkVersion 14
```

Comment:
Bumping the minSdkVerison to 14 for a very specific reason -- I'm going to include an example of using a synchronous version of createToken using rxJava, which requires sdk 14. This is just for the example project, though, so it won't affect anyone using the sdk.